nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -70,12 +66,24 @@ with lib;
       '';
     };
 
-    enableSSL = mkOption {
+    addSSL = mkOption {
       type = types.bool;
       default = false;
       description = "Whether to enable SSL (https) support.";
```

Comment:
Description proposal:

    Whether to enable HTTPS in addition to plain HTTP. This will set defaults
    for <literal>listen</literal> to listen on all interfaces on the
    respective default ports (80, 443).